sound/soc/sof/intel/pci-mtl.c

```diff
@@ -47,6 +47,7 @@ static const struct sof_dev_desc mtl_desc = {
 	.nocodec_tplg_filename = "sof-mtl-nocodec.tplg",
 	.ops = &sof_mtl_ops,
 	.ops_init = sof_mtl_ops_init,
+	.ops_free = hda_ops_free,
```

Comment:
commit messages: why do we have single quotes before Fixes and at the end of the line?

Reply:
@plbossart I had to add those to fix checkpatch complaints. Not sure if it is because of the --strict that I'm using

Reply:
@ranj063, I'm not sure what warning you had, but for me it was complaining only about the short SHA without the single quotes:
```WARNING: Please use correct Fixes: style 'Fixes: <12 chars of sha1> ("<title line>")' - ie: 'Fixes: 064520e8aeaa ("ASoC: SOF: Intel: Add support for MeteorLake (MTL)")'
#8: 
Fixes: 064520e ("ASoC: SOF: Intel: Add support for MeteorLake (MTL)")
```

Using correct length SHA is OK:
```
Fixes: 064520e8aeaa ("ASoC: SOF: Intel: Add support for MeteorLake (MTL)")
```
in ~/.gitconfig
```
[core]
     abbrev = 12
```

Reply:
@ranj063, after checking the PR out, I see that the SHA length is correct. The issue must be that you try to break the Fixes line. Just leave it as one line w/o the single quotes. The tag should not be wrapped.

Reply:
fixed now Thanks @ujfalusi